cmd/connect/config/config.go

```diff
@@ -220,3 +220,13 @@ func updateEndpointFromEnvironment(endpoint config.Endpoint, providerName string
 
 	return endpoint, endpointURL != nil || endpointAPIKey != nil || endpointAPIKeyHeader != nil
 }
+
+func GetMarketmapEndpointFromConfig(cfg config.OracleConfig) (config.Endpoint, error) {
+	for _, provider := range cfg.Providers {
+		if provider.Type == mmtypes.ConfigType {
```

Comment:
🤔 Not sure this will work with [this](https://github.com/skip-mev/connect/blob/7bc5f1553e66a63125f4a356d56d57cfba98d202/cmd/constants/providers.go#L185) or [this](https://github.com/skip-mev/connect/blob/7bc5f1553e66a63125f4a356d56d57cfba98d202/cmd/constants/providers.go#L175) although the latter should go away soon.